src/app/components/TaskForm.tsx

```diff
@@ -33,28 +33,12 @@ export default function TaskForm({ userId, addTask }: TaskFormProps) {
     date: "",
   });
 
-  // function containsArabic(text: string): boolean {
```

Comment:
**Suggestion: delete instead of commetning out, it will live on in version control**